content/docs/command-reference/update.md

```diff
@@ -6,7 +6,7 @@ projects</abbr>, and corresponding [DVC-files](/doc/user-guide/dvc-file-format).
 ## Synopsis
 
 ```usage
-usage: dvc update [-h] [-q | -v] [--rev <commit>] targets [targets ...]
+usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R] targets [targets ...]
```

Comment:
For code blocks we only have up to 72 chars before the horizontal scroll bar appears 🙂 
```suggestion
usage: dvc update [-h] [-q | -v] [--rev <commit>] [-R]
                  targets [targets ...]
```